### DISCUSSION TITLE
Machine Learning Applications

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/Artificial-Intelligence-Lab#16] [Q&A] [answered]
Define Machine Learning? share your point of view on it's applications

**Comment by YNandini60** [comment, marked as answer]
Machine learning: it is a subset of AI, which uses algorithms that learn from data to make predictions
Examples : Image recognition, medical diagnosis, fraud detection etc...,
It's application:
1.Virtual assistant
2.Applicable in social media
3.In eCommerce (customer support, advertising)
4.In Financial Service
